content/applications/sales/sales/products_prices/prices/currencies.rst

```diff
@@ -1,49 +1,239 @@
-=================================
-How to sell in foreign currencies
-=================================
+==================
+Foreign currencies
+==================
+
+With Odoo, pricelists can be used to manage prices in a number of foreign currencies. Specifically,
+Odoo has the ability to work with 167 total currencies. In order to use multiple currencies in Odoo
+*Sales*, the *Accounting* application **must** be installed.
+
+Settings
+========
 
-Pricelists can also be used to manage prices in foreign currencies.
+Once the *Accounting* app is installed, foreign currencies can be viewed, edited, and added to the
+database. To do that, navigate to :menuselection:`Accounting app --> Configuration --> Settings`,
+scroll to the :guilabel:`Currencies` section, and locate the :guilabel:`Main Currency` setting.
 
-* Check *Allow multi currencies* in :menuselection:`Invoicing/Accounting --> Settings`.
-  As admin, you need *Adviser* access rights on Invoicing/Accounting apps.
+.. image:: currencies/main-currency-setting-page.png
+   :align: center
+   :alt: How the main currency feature appears on settings page in Odoo Accounting.
+
+Odoo automatically sets the :guilabel:`Main Currency` as the currency of the country the company is
+based in.
```

Comment:
Arguable guilabel use– I think both work but Im leaning towards not needing it since it is being described in a general sense
```suggestion
Odoo automatically sets the main currency as the currency of the country the company is based in.
```